### DISCUSSION TITLE
post_233608

### DISCUSSION THREAD

**Discussion by cincuranet** [cincuranet/blog-comments#1248] [Comments]
imported 

https://www.tabsoverspaces.com/id/233608

**Comment by cincuranet** [comment]
Imported comment written by _Mike Rosoft_ on _2020-03-20T16:26:11_:

<p>Thanks, this is precisely what I was looking for.</p>

**Comment by cincuranet** [comment]
Imported comment written by _Vasilis Prantzos_ on _2021-02-10T10:01:40_:

<p>Thank you! You solved my issue within minutes</p>

**Comment by gioviopantat** [comment]
:godmode: Thank you very much! This solved my problem perfectly.

**Comment by undercon** [comment]
Great catch! Thanks, you saved me from a very annoying conundrum.

**Comment by cincuranet** [comment]
Happy I could help. :)

**Comment by Zeleksonob** [comment]
Any wat to disable also the [Debug|Release] option it's annoying a well ?
